linera-views-derive/src/lib.rs

```diff
@@ -112,10 +112,12 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
     let mut load_result_quotes = Vec::new();
     let mut rollback_quotes = Vec::new();
     let mut flush_quotes = Vec::new();
+    let mut test_flush_quotes = Vec::new();
     let mut clear_quotes = Vec::new();
     for (idx, e) in input.fields.into_iter().enumerate() {
         let name = e.clone().ident.unwrap();
         let fut = format_ident!("{}_fut", name.to_string());
+        let test_flush_ident = format_ident!("test{}", idx);
```

Comment:
I'd use `delete` or `deleted` everywhere; that tells the reader more than `test`.

Reply:
Ok.